broker/src/broker_server.cpp

```diff
@@ -0,0 +1,525 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * SPDX-FileCopyrightText: 2016-2020 the prplMesh contributors (see AUTHORS.md)
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+
+#include "broker_server.h"
+
+#include <bcl/beerocks_utils.h>
+
+#include <beerocks/tlvf/beerocks_message.h>
+#include <beerocks/tlvf/beerocks_message_broker.h>
+#include <tlvf/ieee_1905_1/tlvVendorSpecific.h>
+
+// System
+#include <cerrno>
+#include <linux/sockios.h>
+#include <sys/uio.h>
+
+#include <easylogging++.h>
+
+namespace beerocks {
+namespace broker {
+
+//////////////////////////////////////////////////////////////////////////////
+////////////////////////// Local Module Definitions //////////////////////////
+//////////////////////////////////////////////////////////////////////////////
+
+// Number of concurrent connections on the server socket
+#define MAX_SERVER_CONNECTIONS 10
```

Comment:
constexptr, and I think the name is wrong:
```suggestion
static constexpr listen_buffer_size = 10;
```
(we can accept infinite connections, but only 10 between two accept() calls).

10 sounds awfully large, but it doesn't matter much.

Reply:
Done